### PR TITLE
10199 add module website_captcha_nogoogle_crm

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by duanyp1991** [comment]
LGTM

**Comment by elicoidal** [comment]
Propose directly to OCA

**Comment by Rona111** [comment]
cc @elicoidal @duanyp1991 @seb-elico @siyuancn

File: website_captcha_nogoogle_crm/static/description/index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan">Odoo CAPTCHA (No Google) for Contact form</h2>
+            <h4 class="oe_slogan">By Elico-Corp</h4>
```
Comment (elicoidal) [inline website_captcha_nogoogle_crm/static/description/index.html]: s/Elico-Corp/Elico Corp

**Comment by noahzaozao** [comment]
LGTM

**Pull request merged**